oxd-server/src/test/java/org/gluu/oxd/server/SetUpTest.java

```diff
@@ -5,34 +5,39 @@
 import io.dropwizard.testing.DropwizardTestSupport;
 import io.dropwizard.testing.ResourceHelpers;
 import org.apache.commons.lang.StringUtils;
+import org.gluu.oxd.common.response.RegisterSiteResponse;
+import org.gluu.oxd.mock.guice.MockAppModule;
+import org.gluu.oxd.server.op.OpClientFactory;
+import org.gluu.oxd.server.persistence.PersistenceService;
+import org.gluu.oxd.server.service.RpService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testng.annotations.AfterSuite;
 import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Guice;
 import org.testng.annotations.Parameters;
-import org.gluu.oxd.common.response.RegisterSiteResponse;
-import org.gluu.oxd.server.persistence.PersistenceService;
-import org.gluu.oxd.server.service.RpService;
 
 /**
  * Main class to set up and tear down suite.
  *
  * @author Yuriy Zabrovarnyy
  * @version 0.9, 21/08/2013
  */
-
 public class SetUpTest {
 
     private static final Logger LOG = LoggerFactory.getLogger(SetUpTest.class);
 
     public static DropwizardTestSupport<OxdServerConfiguration> SUPPORT = null;
 
 
-    @Parameters({"host", "opHost", "redirectUrls"})
+    @Parameters({"host", "opHost", "redirectUrls", "isLoadTest"})
```

Comment:
Lets introduce `SetUpMockServerTest` which pick it up (with code re-use without copy/pastes). It would be nice to not mess between our normal tests and load tests.

Reply:
Done